src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -289,7 +291,15 @@ def complete(self, jobs):
                             logOutput += '\n\n'
                             for exMsg in listWMExceptionStr(condorFilePath):
                                 logOutput += "\n\n%s\n" % exMsg
-                    condorReport.addError("NoJobReport", 99303, "NoJobReport", logOutput)
+
+                            if condorFile == condorLog:
+                                for condorReason in BasicAlgos.findMagicStr(condorFilePath, '<a n="Reason">'):
```

Comment:
You probably need to add `logOutput += '\n\n'` after this for loop. Just to separate the reason from further log lines.
In addition to that, I expect to see `Reason` only once, but this implementation is good.